### ISSUE TITLE
Flag freeze is wrong

### ISSUE THREAD

**Issue by BrunoMine** [hkzorman/advanced_npc#34]
The flag is being used inversely to the real.
Look at the snippets:
https://github.com/hkzorman/advanced_npc/blob/master/npc.lua#L481
https://github.com/tenplus1/mobs_redo/blob/master/api.lua#L2581

**Comment by hkzorman** [comment]
Yes, this can be changed. However it will require changes in many places and have to be done carefully.

What would you prefer, to change the variable name?

I will leave this open until I get to work on this.

**Comment by BrunoMine** [comment]
If this really is completely reversed, you just have to reverse its meaning, however if you really believed what is described in the code, then it is possible that there are errors in your code, and you will need to revise it.

**Comment by BrunoMine** [comment]
Yes, it is ok!
